packages/material-ui/src/InputLabel/InputLabel.test.js

```diff
@@ -35,6 +35,13 @@ describe('<InputLabel />', () => {
     expect(container.firstChild).not.to.have.class(classes.animated);
   });
 
+  it('should forward the asterisk class to AsteriskComponent when required', () => {
+    const { container } = render(<InputLabel required>Foo</InputLabel>);
+    expect(container.querySelector('.MuiFormLabel-asterisk.MuiInputLabel-asterisk')).not.to.equal(
+      null,
+    );
```

Comment:
This will not generate a helpful message when it fails. The other tests do this right by using a `to.have.class` matcher.

```suggestion
    expect(container.firstChild).to.have.class('MuiFormLabel-asterisk');
    expect(container.firstChild).to.have.class('MuiInputLabel-asterisk');
```

Reply:
Changed to `to.have.class` matcher